paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultPaymentMethodVerticalLayoutInteractorTest.kt

```diff
@@ -60,7 +60,7 @@ class DefaultPaymentMethodVerticalLayoutInteractorTest {
                 )
             ),
             formTypeForCode = { FormHelper.FormType.Empty },
-            updateSelection = {},
+            updateSelection = { _, _ -> },
```

Comment:
I think we should add validation to these, to ensure we're passing the right flags, otherwise we're losing (some already mediocre) test coverage.

It's probably best to remove this param `updateSelection` from the test, and create a turbine that's passed to the test context, that we can validate. Happy to pair on it if you'd like.